app/src/main/java/at/aau/edu/wizards/MainActivity.kt

```diff
@@ -14,16 +18,22 @@ import at.aau.edu.wizards.util.permission.PermissionHandler
 private const val DISCOVER_FRAGMENT_TAG = "DISCOVER_FRAGMENT_TAG"
 private const val LOBBY_FRAGMENT_TAG = "LOBBY_FRAGMENT_TAG"
 private const val GAME_BOARD_FRAGMENT_TAG = "GAME_BOARD_FRAGMENT_TAG"
+const val SHARED_PREFERENCE_USERNAME_KEY = "USERNAME"
```

Comment:
that constant is only used in `MainViewModel`. Please move it there and change visibility to private.